### PR TITLE
Migrated test_glusterd_gluster_process_stop_start_cycle

### PR DESCRIPTION
**Pull request by nik-redhat**
Migrated TC [test_glusterd_gluster_process_stop_start_cycle](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_glusterd_gluster_process_stop_start_cycle.py)

Fixes: #515 
Updates: #292 
Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
Test Run:
```
Summary:
+---------------+-------+--------------+
|    Category   | Cases | Pass Percent |
+---------------+-------+--------------+
| nonDisruptive |   0   |      0       |
|   Disruptive  |   1   |    100.0     |
|     Total     |   1   |    100.0     |
+---------------+-------+--------------+
Table:
 test_glusterd_gluster_process_stop_start_cycle
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     dist    |     PASS    | 388.98778772354126 |
|     rep     |     PASS    | 388.2037150859833  |
|     arb     |     PASS    | 387.2255220413208  |
|   dist-rep  |     PASS    | 397.97118973731995 |
|     disp    |     PASS    |  391.165137052536  |
|   dist-arb  |     PASS    | 402.61913418769836 |
|  dist-disp  |     PASS    | 402.9373404979706  |
+-------------+-------------+--------------------+



Framework runtime : 2762.212471485138
```
This is a costly test :sweat_smile:

**Comment by srijan-sivakumar** [comment]
holy smokes!!! 2k seconds!

File: common/ops/gluster_ops/gluster_ops.py [hunk 1]
```diff
@@ -291,3 +291,41 @@ def get_state(self, node: str) -> dict:
...
+
+        Args:
+            node (str): Node on which glusterd process has to be counted.
+
+        Returns:
+            int: Number of glusterd processes running on the node.
+            None: If the command fails to execute.
+        """
+        ret = self.execute_abstract_op_node("pgrep -c glusterd", node,
+                                            False)
```
Comment (srijan-sivakumar) [inline common/ops/gluster_ops/gluster_ops.py:307]: Just curious...do we have any way to differentiate between the fuse process and brick process ?

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

**Pull request merged**